File: jisoo/programmers/level1/나머지가_1이_되는_수_찾기.py
"""
https://school.programmers.co.kr/learn/courses/30/lessons/87389
"""


def solution(n):
  answer = 0
  for i in range(2, n):
    if n % i == 1:
      return i
  return answer


#한줄 코딩
def solution(n):
  return [x for x in range(1,n+1) if n%x==1][0]

File: seongwon/LV1/나머지가1이되는수찾기.js
//넘어가

function solution(n) {
  result = 2147000000;
  for (let i = 2; i < n; i++) {
    if (n % i == 1 && result > i) {
      result = i;
    }
  }
  return result;
}

console.log(solution(10));
